api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java

```diff
@@ -2105,71 +2105,41 @@ private void enableHypertensionProgram() {
     private void enableMentalHealthProgram() {
         configureBasicProgramDashboard(MentalHealthProgram.MENTAL_HEALTH);
 
-        if (config.getCountry().equals(ConfigDescriptor.Country.MEXICO) || config.getCountry().equals(ConfigDescriptor.Country.LIBERIA)) {
-            apps.add(addToMentalHealthDashboardSecondColumn(app(Apps.PHQ9_GRAPH,
-                    "pih.app.phq9.graph.title",
-                    "icon-bar-chart",
-                    null,
-                    null,
-                    objectNode(
-                            "widget", "obsgraph",
-                            "label", "pih.app.phq9.graph.title",
-                            "icon", "icon-bar-chart",
-                            "conceptId", MirebalaisConstants.PHQ9,
-                            "maxResults", "12"
-                    )),
-                    "coreapps", "dashboardwidgets/dashboardWidget"));
-                }
+        if (config.isComponentEnabled(Components.PHQ9_ON_CLINICIAN_DASHBOARD)) {
```

Comment:
Ah, sorry @jmbabazi .... sorry, I didn't specify this more carefully.

I *don't* think we should start making components for each of these... I think the way we had it previously, with the if/thens based on country.

I was just suggesting we follow the pattern of replacing this:

```
app(Apps.PHQ9_GRAPH,
                    "pih.app.phq9.graph.title",
                    "icon-bar-chart",
                    null,
                    null,
                    objectNode(
                            "widget", "obsgraph",
                            "label", "pih.app.phq9.graph.title",
                            "icon", "icon-bar-chart",
                            "conceptId", MirebalaisConstants.PHQ9,
                            "maxResults", "12"
                    )),
                    "coreapps", "dashboardwidgets/dashboardWidget")
```

with this:

```
  graphs.getPHQ9Graph(ExtensionPoints.CLINICIAN_DASHBOARD_SECOND_COLUMN),
                    "coreapps",
                    "dashboardwidgets/dashboardWidget"))

```

I'd like to generally keep "components" have higher-level pieces of functionality when possible (ie, "mental health", "vitals", etc) and for now rely on site/country-based if/thens in the code... ideally we will come up with a better way to turn graphs on/off outside of making code changes, but I think introducing lots of small components will just increase clutter and complexity...  fyi @brandones